app/agents_v2/primary_agent/agent.py

```
@@ -1,91 +1,41 @@
 
-# Import Agent Sparrow reasoning framework
-from app.agents_v2.primary_agent.reasoning import (
-    ReasoningEngine,
-    ReasoningConfig,
-    ReasoningState
-)
+from app.core.rate_limiting.agent_wrapper import wrap_gemini_agent
+from app.agents_v2.primary_agent.schemas import PrimaryAgentState
+from app.agents_v2.primary_agent.tools import mailbird_kb_search, tavily_web_search
+from app.agents_v2.primary_agent.reasoning import ReasoningEngine, ReasoningConfig
+from app.agents_v2.primary_agent.prompts import AgentSparrowV9Prompts
```

Ruff (0.12.2)

`app.agents_v2.primary_agent.prompts.AgentSparrowV9Prompts` imported but unused

Remove unused import: `app.agents_v2.primary_agent.prompts.AgentSparrowV9Prompts`

(F401)